src/proxyvisor.coffee

```diff
@@ -57,9 +60,13 @@ router.get '/v1/assets/:commit', (req, res) ->
 		return res.status(404).send('Not found') if !app
 		getAssetsPath(app.imageId)
 	.then (path) ->
-		getTarArchive(path)
-	.then (archivePath) ->
-		res.sendFile(archivePath)
+		dest = '/tmp/' + app.commit + '.tar'
+		getTarArchive(path,dest)
+	.then (archive) ->
+		archive.on 'finish', ->
+			res.sendFile(dest)
+	.catch (err) ->
+		console.log err
```

Comment:
This should do a `res.send` of some sort as well, otherwise no response will be sent and the request will just hang until it times out